docs/openapi/components/schemas/common/LinkRole.yml

```diff
@@ -20,6 +20,7 @@ properties:
     title: Target
     description: An entry point, within some Web-based protocol.
     type: string
+    format: uri
```

Comment:
spicy... and possibly breaking JSON Schema Form?

Are we sure this works?

Reply:
This should work. Taken from: https://json-schema.org/understanding-json-schema/reference/string.html. I tried using `iri` and got an undefined warning messages when running `npm t`. With `url` i am not getting any warnings.